Elm.gitignore

```diff
@@ -0,0 +1,2 @@
+elm-stuff/*
+!elm-stuff/exact-dependencies.json
```

Comment:
Apologies, I glossed over the quoted text which answers this.

@evancz do you feel this is a good default to have in the template for Elm, to help ensure new projects are setup correctly?